spec/lib/fedex/track_spec.rb

```diff
@@ -59,6 +59,14 @@ module Fedex
         it "raises an invalid tracking number exception" do
           expect{ fedex.track(options).first }.to raise_error(Fedex::InvalidTrackingNumberError)
         end
+
+        it "raises a no-information-available exception" do
+          expect{ fedex.track(options).first }.to raise_error(Fedex::NoTrackingInformationAvailable)
+        end
+
+        it "raises a unable-to-process-request exception" do
+          expect{ fedex.track(options).first }.to raise_error(Fedex::FedexUnableToProcessRequest)
```

Comment:
these tests are not clear, it makes it look like `fedex.track(options).first` triggers all three errors at the same time, which is not useful - why have different errors when you raise them all at once?

can you rewrite this to make it clear that this is not happening?

Reply:
See updated changes.  LMKWYT

Reply:
great, looks good!